decidim-surveys/lib/decidim/surveys/user_responses_serializer.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Surveys
+    class UserResponsesSerializer < Decidim::Exporters::Serializer
+      include Decidim::TranslationsHelper
+
+      # Public: Exports a hash with the serialized data for the user response.
+      def serialize
+        # Returns a csv export of surveys responses only if they have been published.
+        response = resource
+        {
+          id: response.session_token,
+          created_at: response.created_at,
+          ip_hash: response.ip_hash,
+          user_status: (response.decidim_user_id.present? ? "Registered" : "Unregistered"),
```

Comment:
Actually those should be added through I18n . There may be instances ( like Japan ) where "Registered" / "Unregistered" do not make sense.

Reply:
Done: https://github.com/decidim/decidim/pull/15492/commits/f1ec158b4cf8c48c61fed215afe7bdd98cdf71b5